modules/iap/iap.tf

```diff
@@ -108,15 +108,10 @@ resource "helm_release" "iap" {
   }
 }
 
-resource "kubernetes_annotations" "annotation" {
-  api_version = "v1"
-  kind        = "Service"
-  metadata {
-    name      = var.k8s_backend_service_name
-    namespace = var.namespace
-  }
-  annotations = {
-    "beta.cloud.google.com/backend-config" = "{\"default\": \"${var.k8s_backend_config_name}\"}"
-  }
-  depends_on = [helm_release.iap]
-}
\ No newline at end of file
+## TODO: grant permission to specific backend_service
+resource "google_project_iam_member" "project" {
```

Comment:
We should use https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/iap_web_backend_service_iam to add the allowlist for each service.

Reply:
yeah will take in seprate PR.